README.md

```diff
@@ -88,72 +88,91 @@ Make sure the following properties are part of your `package.json`.
 In your main package file, add the following methods and replace
 `your-package-name` with your package name.
 
-```coffeescript
-consumeToolBar: (toolBar) ->
-  @toolBar = toolBar 'your-package-name'
+```js
+let toolBar;
 
-  # Add buttons and spacers here...
+export function consumeToolBar(toolBar) {
+  toolBar = toolBar('your-package-name');
+  // Add buttons and spacers here...
+}
 
-deactivate: ->
-  @toolBar?.removeItems()
+export function deactivate() {
+  if (toolBar) {
+    toolBar.removeItems();
+    toolBar = null;
+  }
+}
 ```
 
 ### Example
 
-```coffeescript
-consumeToolBar: (toolBar) ->
-  @toolBar = toolBar 'your-package-name'
+```js
+let toolBar;
+
+export function consumeToolBar(toolBar) {
+  toolBar = toolBar('your-package-name');
 
-  # Adding button
-  @toolBar.addButton
-    icon: 'octoface'
-    callback: 'application:about'
+  // Adding button
+  toolBar.addButton({
+    icon: 'octoface',
+    callback: 'application:about',
     tooltip: 'About Atom'
+  });
 
-  # Adding spacer
-  @toolBar.addSpacer()
+  // Adding spacer
+  toolBar.addSpacer();
 
-  # Using custom icon set (Ionicons)
-  button = @toolBar.addButton
-    icon: 'gear-a'
-    callback: 'application:show-settings'
-    tooltip: 'Show Settings'
+  // Using custom icon set (Ionicons)
+  const button = toolBar.addButton({
+    icon: 'gear-a',
+    callback: 'application:show-settings',
+    tooltip: 'Show Settings',
     iconset: 'ion'
+  });
 
-  # Disable button
-  button.setEnabled false
+  // Disable button
+  button.setEnabled(false);
 
-  # Function with data in callback
-  @toolBar.addButton
+  // Function with data in callback
+  toolBar.addButton({
     icon: 'alert',
-    callback: (data)->
-      alert data
-    tooltip: 'Show Alert'
+    callback(data) {
+      alert(data);
+    },
+    tooltip: 'Show Alert',
     data: 'foo'
+  });
 
-  # Callback with modifiers
-  @toolBar.addButton
+  // Callback with modifiers
+  toolBar.addButton({
     icon: 'octoface',
-    callback:
-      '': 'application:cmd-1'  # Without modifiers is default action
-      'alt': 'application:cmd-2'
-      'ctrl': 'application:cmd-3'
-      'shift': (data) -> console.log data  # With function callback
-      'alt+shift': 'application:cmd-5'  # Multiple modifiers
-      'alt+ctrl+shift': 'application:cmd-6'  # All modifiers      
+    callback: {
+      '': 'application:cmd-1',      // Without modifiers is default action
+      'alt': 'application:cmd-2',
+      'ctrl': 'application:cmd-3',  // With function callback
+      'shift'(data) {
```

Comment:
Short methods can be computed, string literals or just identifiers. Here, you don't need the quotes for `alt`, `ctrl`, or `shift`, but I kept them so the entire object looked consistent. I can remove them though.